Framework/Geometry/src/Surfaces/Plane.cpp

```diff
@@ -306,14 +306,24 @@ int Plane::LineIntersectionWithPlane(V3D startpt, V3D endpt, V3D &output) {
   double sprod = this->getNormal().scalar_prod(startpt - endpt);
   if (sprod == 0)
     return 0;
-  double s1 = (NormV[0] * startpt[0] + NormV[1] * startpt[1] +
-               NormV[2] * startpt[2] - Dist) /
-              sprod;
+  double projection =
```

Comment:
Thank you for improving names and breaking up the calculation. You could `const` these to allow the compiler to do its magic.

Reply:
done